website/docs/deployment.mdx

```diff
@@ -90,41 +90,41 @@ export default function Home() {
 
 There are a few common hosting options:
 
-- [Self hosting](#self-hosting) with an HTTP server like Apache2 or Nginx;
-- Jamstack providers, e.g. [Netlify](#deploying-to-netlify) and [Vercel](#deploying-to-vercel). We will use them as references, but the same reasoning can apply to other providers.
-- [GitHub Pages](#deploying-to-github-pages). (By definition, it is also Jamstack, but we compare it separately.)
+- [Self hosting](#self-hosting) with an HTTP server like Apache2 or Nginx
+- Jamstack providers (e.g. [Netlify](#deploying-to-netlify) and [Vercel](#deploying-to-vercel)). We will use them as references, but the same reasoning can apply to other providers.
+- [GitHub Pages](#deploying-to-github-pages) (By definition, it is also Jamstack, but we compare it separately).
 
 If you are unsure of which one to choose, ask the following questions:
 
 <details>
 
 <summary>
-  How much resource (person-hours, money) am I willing to invest in this?
+  How much time and money am I willing to invest in this?
 </summary>
 
-- 🔴 Self-hosting is the hardest to set up—you would usually need an experienced person to manage this. Cloud services are almost never free, and setting up an on-site server and connecting it to the WAN can be even more costly.
-- 🟢 Jamstack providers can help you set up a working website in almost no time and offers features like server-side redirects that are easily configurable. Many providers offer generous build time quotas even for free plans that you would almost never exceed. However, it's still ultimately limited—you would need to pay once you hit the limit. Check the pricing page of your provider for details.
-- 🟡 The GitHub Pages deployment workflow can be tedious to set up. (Evidence: see the length of [Deploying to GitHub Pages](#deploying-to-github-pages)!) However, this service (including build and deployment) is always free for public repositories, and we have detailed instructions to help you make it work.
+- 🔴 Self-hosting requires experience in networking as well as Linux and web server administration. It's the most difficult option, and would require the most time to manage successfully. Expense-wise, cloud services are almost never free, and purchasing/deploying an onsite server can be even more costly.
+- 🟢 Jamstack providers can help you set up a working website in almost no time and offers features like server-side redirects that are easily configurable. Many providers offer generous build-time quotas even for free plans that you would almost never exceed. However, free plans have limits, and you would need to pay once you hit those limits. Check the pricing page of your provider for details.
+- 🟡 The GitHub Pages deployment workflow can be tedious to set up (Evidence: see the length of [Deploying to GitHub Pages](#deploying-to-github-pages)!). However, this service (including build and deployment) is always free for public repositories, and we have detailed instructions to help you make it work.
```

Comment:
The parentheses here enclose a complete sentence (technically not, but semantically so), so it's right to put the period before.

```suggestion
- 🟡 The GitHub Pages deployment workflow can be tedious to set up. (Evidence: see the length of [Deploying to GitHub Pages](#deploying-to-github-pages)!) However, this service (including build and deployment) is always free for public repositories, and we have detailed instructions to help you make it work.
```